app/build.gradle

```diff
@@ -122,6 +122,8 @@ dependencies {
     implementation 'com.android.support.constraint:constraint-layout:1.0.2'
     implementation 'com.crashlytics.sdk.android:crashlytics:2.9.1'
     implementation 'com.squareup.picasso:picasso:2.71828'
+    implementation 'de.hdodenhof:circleimageview:2.2.0'
```

Comment:
We already had that library in the app, see line 107

Reply:
Oops, fixed